weather_mv/loader_pipeline/netcdf_loader.py

```diff
@@ -48,23 +49,48 @@ def configure_logger(verbosity: int) -> None:
     logger.setLevel(level)
 
 
-def open_dataset(uri: str) -> xr.Dataset:
+def __open_dataset_file(filename: str) -> xr.Dataset:
+    try:
+        return xr.open_dataset(filename)
+    except ValueError as e:
+        e_str = str(e)
+        if not ("Consider explicitly selecting one of the installed engines" in e_str and "cfgrib" in e_str):
+            raise
+    # Trying with explicit engine for cfgrib.
+    try:
+        return xr.open_dataset(filename, engine='cfgrib', backend_kwargs={'indexpath': ''})
+    except ValueError as e:
+        if not "multiple values for key 'edition'" in str(e):
+            raise
+
+    # Try with edition 1
+    # Note: picking edition 1 for now as it seems to get the most data/variables for ECMWF realtime data.
```

Comment:
edition 1 is opinionated, even within ECMWF data. Since real-time dissemination we've experienced has been grib 1, i'm ok with this. However, please add a log message at the warning level to help inform the user of this choice. 

we don't want users running this on historical files wondering why data is being dropped with no warning.